src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -272,12 +278,94 @@ private static Process StartNewVisualStudioProcess(string installationPath)
             IntegrationHelper.KillProcess("dexplore");
 
             var process = Process.Start(vsExeFile, VsLaunchArgs);
-
             Debug.WriteLine($"Launched a new instance of Visual Studio. (ID: {process.Id})");
 
             return process;
         }
 
+        // Starts ProcDump against the processId.
+        // Returns the path to the dump directory.
+        private static string StartProcDump(int processId)
```

Comment:
thanks! simplified

---
In reply to: [121030977](https://github.com/dotnet/roslyn/pull/20129#discussion_r121030977) [](ancestors = 121030977)